js/forms.js

```diff
@@ -1,3 +1,5 @@
+import { resetScale } from './scale.js';
+import { resetEffects } from './effect.js';
```

Comment:
ошибка в названии файла 
```suggestion
import { resetScale } from './scale.js';
import { resetEffects } from './effects.js';
```